lib/testing/src/lib/process-services-cloud/pages/task-header-cloud-component.page.ts

```diff
@@ -77,4 +80,20 @@ export class TaskHeaderCloudPage {
         return BrowserActions.getText(this.dueDateField);
     }
 
+    checkDescriptionEditIconIsDisplayed() {
```

Comment:
We can just keep 1 method for isDisplayed, and 1 method for isNotDisplayed, and send the element name dynamically for assertions.

Reply:
From my point of view, it's better to have methods for each property, because it's much more clear for devs that will create new tests which are the properties of task header component. Otherwise we will have to provide the element name from the tests, which is not ideal. In case something changes, all tests will be affected.